actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -17,6 +17,7 @@ class Railtie < Rails::Railtie # :nodoc:
     config.action_dispatch.encrypted_cookie_salt = 'encrypted cookie'
     config.action_dispatch.encrypted_signed_cookie_salt = 'signed encrypted cookie'
     config.action_dispatch.perform_deep_munge = true
+    config.action_dispatch.asset_headers = { }
```

Comment:
This sounds like headers on the asset pipeline rather than headers on static file response.

The other config scattered among `config.static_cache_control` and `config.serve_static_files`. Let's unify these as "public file server" config, reflecting that we're just serving files from the highly visible `public/` dir:
```ruby
config.public_file_server.enabled = true  # replace config.serve_static_files
config.public_file_server.headers = { 'Cache-Control' => '…' }  # replace cache.static_cache_control
```